spec/lib/miq_expression_spec.rb

```diff
@@ -510,7 +510,7 @@
     it "generates the SQL for multi level contains with a scope" do
       sql, _ = MiqExpression.new("CONTAINS" => {"field" => "ExtManagementSystem.clustered_hosts.operating_system-name", "value" => "RHEL"}).to_sql
       rslt = "\"ext_management_systems\".\"id\" IN (SELECT \"ext_management_systems\".\"id\" FROM \"ext_management_systems\" " \
-             "INNER JOIN \"hosts\" ON \"hosts\".\"ems_id\" = \"ext_management_systems\".\"id\" AND \"hosts\".\"ems_cluster_id\" IS NOT NULL " \
+             "INNER JOIN \"hosts\" ON \"hosts\".\"ems_cluster_id\" IS NOT NULL AND \"hosts\".\"ems_id\" = \"ext_management_systems\".\"id\" " \
```

Comment:
I really like how you broke up the other one into `<<~EXPECTED`...does it make sense to do that here too?

Reply:
Was trying to avoid making too many changes, and that one specifically was hard to grok in the errors from rspec, so doing that allowed me to see it easier.

That said, not opposed to the idea, was just avoiding making too many large changes when I could to reduce the number of "variables" changed in a single PR.

Reply:
Up to you...I have no real preference in this PR.

Reply:
I will wait then, but leave this comment as unresolved as a secondary reminder that I do want to do it as a follow up.